.pre-commit-config.yaml

```diff
@@ -25,6 +25,16 @@ repos:
         entry: pylint
         args: [--rcfile=tests/garage/.pylintrc]
 
+-   repo: https://github.com/timothycrosley/isort
```

Comment:
did the official hook end up not being very good?

Reply:
I think this is the official one. The pre-commit one is depreciated: https://github.com/pre-commit/mirrors-isort